.travis.yml

```diff
@@ -8,17 +8,27 @@ script: skip
 
 jobs:
     include:
-        - stage: test
-          php: 7.2
-          name: Run bin
-          script:
-              - bin/CIConfigGen.php
+        -
+            stage: test
+            php: 7.2
+            name: Run bin
+            script:
+                - bin/CIConfigGen.php
+                - bin/CIConfigGen.php craft:generate composer.json
```

Comment:
Dependency file is now optional argument. If nothing is provided, root `composer.json` is used